src/screens/library_screen/library_screen.tsx

```diff
@@ -77,6 +86,10 @@ export class LibraryScreen extends Component<ownProps,ownState> {
         this.props.navigation.navigate('Search')
     }
 
+    handleToggleLibrarySearch = () => {
+        this.setState({ openSearchInput: !this.state.openSearchInput })
```

Comment:
At some point the renders on react would be completely asynchronous so tobe sure please use this code instead
`this.setState((currentState) => ({openSearchInput: !currentState.openSearchInput})`